src/Serializer/AbstractItemNormalizer.php

```diff
@@ -97,9 +100,25 @@ public function normalize($object, $format = null, array $context = [])
             $context['resources'][$resource] = $resource;
         }
 
+        if ($this->isItemSerializationShouldBeSkipped($context)) {
+            return $context['iri'] ?? $this->iriConverter->getIriFromItem($object);
+        }
+
         return parent::normalize($object, $format, $context);
     }
 
+    /**
+     * Is normalization should be skipped on this item ?
+     *
+     * @param array $context
+     *
+     * @return bool
+     */
+    protected function isItemSerializationShouldBeSkipped(array $context)
```

Comment:
`shouldSkipSerialization` same you can add a return type `:bool` and remove `@param/@return`.